recipe/meta.yaml

```
@@ -36,20 +36,24 @@ test:
   imports:
     - yarl
   source_files:
+    - pytest.ini
```

```suggestion
    - pytest.ini
```
At a quick glance, this doesn't seem necessary. Do the tests fail without it?

---

Not presently but since it's a pytest config file and we're running the tests with pytest I considered it a bit of future-proofing to include in the test environment.